pkg/check/node_cluster_permissions.go

```diff
@@ -50,7 +56,14 @@ func (c *CheckComputeClusterPermissions) checkComputeClusterPrivileges(ctx *Chec
 
 func (c *CheckComputeClusterPermissions) CheckNode(ctx *CheckContext, node *v1.Node, vm *mo.VirtualMachine) error {
 	var errs []error
-	err := c.checkComputeClusterPrivileges(ctx, vm)
+	readOnly := false
+
+	// If pre-existing resource pool was defined, only check cluster for read privilege
+	if ctx.VMConfig.Workspace.ResourcePoolPath != "" {
```

Comment:
Is this variable set for a cluster default resource pool, or only if another resource pool is explicitly defined?